src/App.tsx

```diff
@@ -1,14 +1,80 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { getTodos, getUser } from './api';
+
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { Options } from './types/Options';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { filteredTodos } from './components/utils';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [user, setUser] = useState<User | null>(null);
+  const [currentTodo, setcurrentTodo] = useState<Todo | null>(null);
+  const [isDataReady, setIsDataReady] = useState<boolean>(false);
```

Comment:
You don't really need to specify a type here if you can only have boolean values in this state. So, you can remove it
```suggestion
  const [isDataReady, setIsDataReady] = useState(false);
```

Reply:
Don't forget to load [user details](https://mate-academy.github.io/react_dynamic-list-of-todos/api/users/1.json) (replace `1` with the actual `userId`);
1. Show the Loader while waiting for the user;

Could you sync your corrections with read.me? Its makes me extremely confusing